src/main/java/org/spongepowered/common/mixin/core/data/MixinDataHolder.java

```diff
@@ -413,9 +413,4 @@ public DataContainer toContainer() {
         return Collections.emptyList();
     }
 
-    @Override
-    public DataHolder copy() {
```

Comment:
Sorry, you're right. I was referring to moving the implementation from `MixinDataHolder` to more specific mixins, but didn't realize there was a better solution. Will fix later today.